contracts/moda-contracts/src/interfaces/IModaRegistry.sol

```diff
@@ -2,47 +2,67 @@
 pragma solidity 0.8.21;
 
 interface IModaRegistry {
+    event CatalogMembershipChanged(string name, address indexed membership);
+    event CatalogRegistered(string name, address indexed catalog);
+    event CatalogUnregistered(string name, address indexed catalog);
+
+    event ManagerAdded(address indexed artist, address indexed manager);
+    event ManagerRemoved(address indexed artist, address indexed manager);
+
+    event TreasuryFeeChanged(uint256 oldFee, uint256 newFee);
+    event TreasuryChanged(address oldTreasury, address newTreasury);
+
     struct Catalog {
         string name;
-        address catalogAddress;
+        address catalog;
+        address membership;
     }
 
     /// Membership
 
     /**
      * @dev Checks the user is registered to a particular catalog
```

Comment:
"Checks if the user is a member of a particular catalog. This is a proxy to the Membership contract instance deployed by the catalog owner."